extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlPortForward.java

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl;
+
+import static io.kubernetes.client.extended.kubectl.KubectlExec.copyAsync;
+
+import io.kubernetes.client.PortForward;
+import io.kubernetes.client.extended.kubectl.exception.KubectlException;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+
+public class KubectlPortForward
+    extends Kubectl.ResourceAndContainerBuilder<Boolean, KubectlPortForward>
```

Comment:
```suggestion
    extends Kubectl.ResourceAndContainerBuilder<V1Pod, KubectlPortForward>
```

is it majorly targeting pod resource? the first type argument is for specifying which kind of kubernetes resource is the resource builder addressing

Reply:
done.